model/schema/controlledVocabulary.yaml

```diff
@@ -91,10 +91,15 @@ classes:
       - name
       - vocabulary_description
       - member_terms
+      - synonyms
     slot_usage:
       name:
         required: true
         identifier: true
+      synonyms:
+        description: >-
+          Free text synonym(s) of a controlled vocabulary term
```

Comment:
is this supposed to be on "Vocabulary"?  the description talks about vocabulary `term` ?

Reply:
@sierra-moxon Good catch! Thanks! Fixed